applications/thingy91/usb_uart_bridge/src/main.c

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
@carlescufi Is this licensing okay, or should it be the BSD-5 thing?

Reply:
Changed to BSD-5-clause to bring it in line with the other samples